src/pages/Swipe/Swipe.js

```diff
@@ -1,5 +1,58 @@
-function Swipe() {
+import React, { useState, useMemo, useRef } from 'react'
+import { useNavigate } from 'react-router-dom';
+import Header from '../../components/Header/Header';
+import Card from '../../components/Card/Card';
+import './Swipe.css'
+import './components/FoodCard/FoodCard.js'
+import FoodCard from './components/FoodCard/FoodCard.js';
+
+  
+export default function Swipe() {
+  const navigate = useNavigate();
+
+  const cards = [
+    {
+      image: 'https://img.icons8.com/color/452/GeeksforGeeks.png',
+      color: '#55ccff'
+    },
+    {
+      image: 'https://img.icons8.com/color/452/GeeksforGeeks.png',
+      color: '#e8e8e8'
+    },
+    {
+      image: 'https://img.icons8.com/color/452/GeeksforGeeks.png',
+      color: '#0a043c'
+    },
+    {
+      image: 'https://img.icons8.com/color/452/GeeksforGeeks.png',
+      color: 'black'
+    }
+  ];
+
+    return (
+        <>
+          <Header/>
+          <div className='pagey'>
+              <h1 className='heading'>Swipe<span>Ripe</span></h1>
```

Comment:
your classnames should be very specific. Since you are working on the swipe page, name it "swipe-heading". This is because if others were to use general classnames like heading for styling, your CSS styles will be inherited and apply to their elements. Then they will be confused on why there styles don't apply and not know that it is coming from your files